src/lib/core/tt_compression.c

```diff
@@ -0,0 +1,14 @@
+/*
+ * SPDX-License-Identifier: BSD-2-Clause
+ *
+ * Copyright 2010-2021, Tarantool AUTHORS, please see AUTHORS file.
+ */
+#include "trivia/config.h"
+
+#if defined(ENABLE_TUPLE_COMPRESSION)
+# error unimplemented
+#endif
+
+const char *compression_type_strs[] = {
+        /* [COMPRESSION_TYPE_NONE]       = */ "none",
```

Comment:
Nit: you don't need to comment `[COMPRESSION_TYPE_NONE] = `.

Reply:
fixed